packages/babel-core/test/fixtures/plugins/multiple-definition-evaluation/exec.js

```diff
@@ -1,8 +1,8 @@
-var code = multiline([
```

Comment:
nit: Can we also remove `multiline` from `packages/babel-helper-transform-fixture-test-runner/src/helpers.js`? It is never exported from `index.js`.

Reply:
Not without a major bump. It's installed into the exec context, so it's available to tests. It's probably unused.

Reply:
If it's a concern around people using `babel-helper-transform-fixture-test-runner` as a dep I think we've never supported that as an API so it's fine (https://www.npmjs.com/package/@babel/helper-transform-fixture-test-runner). It's internal only essentially (we have no docs on it either), not that it's a good thing